js/samples/04.ai-apps/e.assistants-orderBot/README.md

```diff
@@ -100,12 +100,12 @@ If you are using OpenAI then follow these steps:
 ```
 
 1. Ensure you have downloaded and installed [Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
-2. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
-3. Copy this sample into a new folder outside of teams-ai
-4. Select File > Open Folder in VS Code and choose this sample's directory
-5. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
-6. Ensure that you have set up the sample from the previous step.
-7. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
-8. In the browser that launches, select the **Add** button to install the app to Teams.
+1. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
```

Comment:
not sure how this happened but this all became 1's 😆

Reply:
This is a markdown feature - it will sequentially bump the number if all of the list is 1's. Good practice for in case we add or delete lines from the list in the future.